APCAppCore/APCAppCore/UI/Onboarding/APCShareViewController.m

```diff
@@ -307,12 +307,11 @@ - (void)messageComposeViewController:(MFMessageComposeViewController *)controlle
 
 - (IBAction) okayTapped: (id) __unused sender
 {
-    if (self.taskViewController != nil) {
+    if (self.goBackIfUserHitsOkay) {
+        [self back];
+    } else if (self.taskViewController != nil) {
```

Comment:
Not quite. This will ONLY work if using the AppCore onboarding which works differently from the onboarding we moved to for mPower which uses an ORKTaskViewController and *not* a navigation controller as is assumed by the `[self back]` method.  Additionally, check with Christine, I thought I remembered that she wanted the behavior to be that the view controller is dismissed.